.gitignore

```diff
@@ -0,0 +1,46 @@
+*.py[cod]
+
+# C extensions
+*.so
+
+# Packages
+*.egg
+*.egg-info
+dist
+build
+eggs
+parts
+bin
+var
+sdist
+develop-eggs
+.installed.cfg
+lib
+lib64
+
+# Installer logs
+pip-log.txt
+
+# Unit test / coverage reports
+.coverage
+.tox
+nosetests.xml
+htmlcov
+
+# Integration test configuration
+integration_tests/config.py
+
+# Translations
+*.mo
+
+# Mr Developer
```

Comment:
It's a bit weird seeing dev tools that aren't actually used in the project being gitignored. Maybe strip it down?